Css canvas medical - YoonHwan Cho

YoonHCho opened the pull request:
Css canvas medical - YoonHwan Cho
![CSS Canvas Medical](https://user-images.githubusercontent.com/106636383/177056092-713382aa-0813-4fb5-a687-fb8645346620.gif)

YoonHCho commented:
Completed practice exercise css canvas product
![CSS Canvas Product](https://user-images.githubusercontent.com/106636383/177247751-8385ff01-4bae-49c0-b79e-aaf1fe8e1bf2.gif)

codingcodymiller reviewed (changes requested):
Hey Yoon, your fonts are wrong in multiple different places, I see uses of an incorrect font family, wrong font sizes, and possibly some incorrect font-weight and font-style. Please revise this and submit a new screenshot with your revision.

YoonHCho commented:
removed the address element and the italic from it. changed the font style on the headers and sizing on some
![CSS Canvas Medical](https://user-images.githubusercontent.com/106636383/177470812-eb44150d-75c7-43f4-9f70-a95b6b91731b.gif)

YoonHCho commented:
removed the address element and the italic from it. changed the font style on the headers and sizing on some
![2022-07-06 13_02_00-JavaScript Data Model](https://user-images.githubusercontent.com/106636383/177647628-fe4d6630-e87f-4acd-a109-e5f667ae4f46.png)

codingcodymiller commented on css-canvas-medical/index.html:
```diff
@@ -0,0 +1,124 @@
+
+    <hr>
+
+    <div class="spacing"></div>
```
You shouldn't use empty elements for spacing, you should be using `margin` or `padding`

codingcodymiller commented on css-canvas-medical/index.html:
```diff
@@ -0,0 +1,124 @@
+        </div>
+      </div>
+    </section>
+    <div class="spacing"></div>
```
You shouldn't use empty elements for spacing, you should be using `margin` or `padding`

YoonHCho commented:
Hi Cody, removed the spacing and used the margin-top and bottom
![CSS Canvas Medical](https://user-images.githubusercontent.com/106636383/177856749-1eca6b6c-687c-40a6-b767-4200f29c21da.gif)

codingcodymiller reviewed (approved):
Great work!!

YoonHCho commented:
completed canvas practice product.html & product.css
![CSS Canvas product](https://user-images.githubusercontent.com/106636383/178120648-20d085f6-09f1-456a-a530-95a7291f50f0.gif)

YoonHCho commented:
completed canvas practice implementation.html & implementation.css

![CSS Canvas implementation](https://user-images.githubusercontent.com/106636383/178126066-9edb423b-fcb1-489c-bef6-21a50dd816ce.gif)
